### PR TITLE
Doc: Correct documentations by ChineseLinter and fix typos.

### PR DESCRIPTION
**Pull request by xylinkun**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [x] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### CODE REVIEW

File: docs/cn/layers.md [hunk 1]
```diff
@@ -57,81 +57,81 @@ Vim 插件以及相关配置。而 SpaceVim 是以模块的方式来组织和管
...
-| [shell](shell/)    |   这一模块为 SpaceVim 提供了终端集成特性，优化内置终端的使用体验 | 
-| [sudo](sudo/)    |   sudo 提供了在 SpaceVim 中以管理员身份读写文件的功能。 | 
-| [tags](tags/)    |   tags 模块提供全局的 tags 索引管理，提供快速检索定义和引用的功能。 | 
-| [tmux](tmux/)    |   该模块主要提供了一些在 Vim 内操作 tmux 的功能，使得在 tmux 窗口之间跳转更加便捷。 | 
-| [tools#dash](tools/dash/)    |   该模块提供对 Dash 支持，可快速查找光标位置的单词 | 
-| [tools](tools/)    |   集成多种常用工具，包括日历、计算器、等等多种工具类插件，该模块针对 vim8 以及 neovim 提供了更好的插件选择。 | 
-| [ui](ui/)    |   SpaceVim ui 模块提供了一个 IDE-like 的界面，包括状态栏、文件树、语法数等等特性。 | 
+| 名称                                  | 描述                                                                                                                                |
+| ------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| [VersionControl](VersionControl/)     | 这一模块为 SpaceVim 提供了通用的代码版本控制支持，该模块支持 Git、Mercurial、Bazaar、SVN 等等多种后台工具。                         |
```
Comment (wsdjeg) [inline docs/cn/layers.md]: layer 列表是自动生成的，包括描述，内容取自每个layer文档里的 ·description:·

Comment (xylinkun) [reply docs/cn/layers.md]: 那还需要改吗？我是觉得各模块的描述参差不齐，文档看起来挺丑的。

Comment (wsdjeg) [reply docs/cn/layers.md]: 这个要是改的话，需要改每一个模块的描述，然后在 `layers.md` 这个文档里执行：

`call SpaceVim#dev#layers#updateCn()`

**Comment by wsdjeg** [comment]
Thanks, merged.

File: README.md [hunk 2]
```diff
@@ -177,14 +176,14 @@ Bitcoin: 1DtuVeg81c2L9NEhDaVTAAbrCR3pN5xPFv
 
 ### Credits & Thanks
 
-This project exists thanks to all the people who have [contributed](CONTRIBUTING.md):
+This project exists to thank all the people who have [contributed](CONTRIBUTING.md):
```
Comment (bl-ue) [inline README.md:179]: This is wrong...

File: README.md [hunk 3]
```diff
@@ -159,13 +159,12 @@ For more features, please read [SpaceVim's Blog](https://spacevim.org/blog/)
 
 ### Support SpaceVim
 
-The best way to support SpaceVim is to contribute to it either by reporting bugs,
-helping the community on the [Gitter Chat](https://gitter.im/SpaceVim/SpaceVim) or sending pull requests.
+The best way to support SpaceVim is to contribute to it either by reporting bugs.
+Helping the community on the [Gitter Chat](https://gitter.im/SpaceVim/SpaceVim) or sending pull requests.
```
Comment (bl-ue) [inline README.md:163]: This is wrong...

File: README.md (hunk 2 above)
Comment (bl-ue) [reply README.md:179]: This was fixed in `master`, I see.

File: README.md (hunk 3 above)
Comment (bl-ue) [reply README.md:163]: This is still in `master`, to this day.

**Pull request merged**